File: boot-mybatis-plus/src/main/java/com/dxp/mybatisplus/service/impl/UserServiceImpl.java
package com.dxp.mybatisplus.service.impl;

import com.baomidou.mybatisplus.extension.service.impl.ServiceImpl;
import com.dxp.mybatisplus.entity.User;
import com.dxp.mybatisplus.mapper.UserMapper;
import com.dxp.mybatisplus.service.UserService;
import org.springframework.stereotype.Service;

/**
 * @author carzy
 * @date 2020/6/28
 */
@Service
public class UserServiceImpl extends ServiceImpl<UserMapper, User> implements UserService {
}

File: boot-shiro/src/main/java/com/dxp/shiro/web/service/UserServiceImpl.kt
package com.dxp.shiro.web.service

import com.dxp.shiro.common.Constants
import com.dxp.shiro.web.entity.sys.User
import org.apache.shiro.crypto.hash.Md5Hash
import org.springframework.stereotype.Service

@Service
class UserServiceImpl : UserService {

    override fun findByUsername(name: String?): User? {
        val md5 = Md5Hash("12345678", null, Constants.MD5_HASH_ITERATIONS)
        //todo.. 查数据库
        val user = User();
        user.id = 1L
        user.username = "test"
        user.password = md5.toString()
        return user
    }
}